package.json

```diff
@@ -42,6 +42,7 @@
     "standard": "^14.0.0"
   },
   "dependencies": {
+    "@waff/yggdrasil": "^1.1.4",
```

Comment:
> That's why we use a specific version of a library (which code we've reviewed) and not blindly the latest version.

shouldn't it be `1.1.4` rather then `^1.1.4 := >=1.1.4 <2.0.0`

Reply:
no, that's how semver works